apps/console/src/extensions/extension.tsx

```diff
@@ -50,7 +50,11 @@ export const ComponentPlaceholder = (props: ExtensionInterface): ReactElement =>
     let DynamicLoader;
 
     if (fragment) {
-        DynamicLoader = lazy(() => import(`${fragment}`));
+        DynamicLoader = lazy(() => {
+            return new Promise(resolve => {
+              setTimeout(() => resolve(import(`${fragment}`)), 500);
```

Comment:
Won't this delay the appearance of the extension by `500` ms?

Reply:
Removed this logic